### PR TITLE
Add system memory hint for memory advise

### PR DESCRIPTION
**Pull request by jandres742**
Resolves: #133

### CODE REVIEW

**Comment by jandres742** [comment]
@wdamon-intel , @michalmrozek, @smilczar: 
Please review.

File: scripts/core/copy.yml [hunk 1]
```diff
@@ -459,6 +459,12 @@ etors:
       desc: "hints that memory should be cached"
     - name: BIAS_UNCACHED
       desc: "hints that memory should be not be cached"
+    - name: SET_SYSTEM_MEMORY_PREFERRED_LOCATION
+      version: "1.7"
+      desc: "hint that the preferred memory location is the host"
```
Comment (MichalMrozek) [inline scripts/core/copy.yml:464]: hint that the preferred memory location is host memory

Comment (jandres742) [reply scripts/core/copy.yml:464]: thanks. changing.

**Review by wdamon-intel — APPROVED** [review]
LGTM

**Review by MichalMrozek — APPROVED** [review]

File: scripts/core/copy.yml [hunk 2]
```diff
@@ -459,6 +459,12 @@ etors:
       desc: "hints that memory should be cached"
     - name: BIAS_UNCACHED
       desc: "hints that memory should be not be cached"
+    - name: SET_SYSTEM_MEMORY_PREFERRED_LOCATION
+      version: "1.7"
+      desc: "hint that the preferred memory location is host memory"
+    - name: CLEAR_SYSTEM_MEMORY_PREFERRED_LOCATION
+      version: "1.7"
+      desc: "removes the affect of $X_MEMORY_ADVICE_SET_SYSTEM_MEMORY_PREFERRED_LOCATION"
```
Comment (MichalMrozek) [inline scripts/core/copy.yml:467]: typo -> affect

**Pull request merged**